apps/emqx_management/src/emqx_mgmt_api_status.erl

```diff
@@ -45,7 +45,12 @@ running_status() ->
             BrokerStatus = broker_status(),
             AppStatus = application_status(),
             Body = io_lib:format("Node ~ts is ~ts~nemqx is ~ts", [node(), BrokerStatus, AppStatus]),
-            {200, #{<<"content-type">> => <<"text/plain">>}, list_to_binary(Body)};
+            StatusCode =
+                case AppStatus of
+                    running -> 200;
+                    not_running -> 503
+                end,
+            {StatusCode, #{<<"content-type">> => <<"text/plain">>}, list_to_binary(Body)};
```

Comment:
should we add retry-after ? like the false clause below.

Reply:
:heavy_check_mark: